packages/fiori/src/Wizard.ts

```diff
@@ -493,7 +507,10 @@ class Wizard extends UI5Element {
 
 		this._prevWidth = this.width;
 		this._prevContentHeight = this.contentHeight;
-		this._breakpoint = RESPONSIVE_BREAKPOINTS[Object.keys(RESPONSIVE_BREAKPOINTS).findLast(size => Number(size) < this.width)] || RESPONSIVE_BREAKPOINTS["0"];
+
+		const breakpointDimensions = Object.keys(RESPONSIVE_BREAKPOINTS).reverse();
```

Comment:
Chance to refactor a bit and extract line511-512-513 in a method - (getCurrentBreakpoint, getBreakpoint, calcCurrentBreakpoint, nextBreakpoint)